src/errors.ts

```diff
@@ -274,7 +274,6 @@ export const errorMessages: {
   [Errors.IllegalReturn]: 'Illegal return statement',
   [Errors.InvalidForLHSBinding]: 'The left hand side of the for-header binding declaration is not destructible',
   [Errors.InvalidNewTarget]: 'new.target only allowed within functions',
-  [Errors.InvalidEscapedKeyword]: "'Unexpected token: 'escaped keyword'",
```

Comment:
This is a duplicated entry of line 207: `[Errors.InvalidEscapedKeyword]: "Unexpected token: 'escaped keyword'"`